include/cutlass/numeric_conversion.h

```diff
@@ -1148,8 +1148,18 @@ struct NumericArrayConverter<cutlass::bfloat16_t, float, 2, FloatRoundStyle::rou
 
     unsigned d;
 
+  #if defined __CUDA_ARCH__ && __CUDA_ARCH__ >= 800 || defined __SYCL_CUDA_ARCH__ && __SYCL_CUDA_ARCH__ >= 800
+
     asm("cvt.rn.bf16x2.f32 %0, %1, %2;\n" : "=r"(d) : "f"(source[1]), "f"(source[0]) );
 
+  #elif defined SYCL_INTEL_TARGET
+
+    auto out = __spirv_ConvertFToBF16INTEL(__ocl_vec_t<float, 2>{source[1], source[0]});
```

Comment:
@rolandschulz mentioned that: For AOT this has been fixed: https://github.com/intel/llvm/pull/16494 . Do we want to keep the workaround?

Reply:
Fixed this.